src/index.html

```diff
@@ -140,5 +140,9 @@ <h2 class="one-recipe-title">
     <script type="text/javascript" src="../data/recipes.js"></script>
     <script type="text/javascript" src="../data/ingredients.js"></script>
     <script type="text/javascript" src="./scripts.js"></script>
+    <script type="text/javascript" src="./User.js"></script>
+    <script type="text/javascript" src="./Pantry.js"></script>
+    <script type="text/javascript" src="./Recipe.js"></script>
+    <script type="text/javascript" src="./scripts.js"></script>
```

Comment:
This line is duplicated at 142, we should probably get rid of line 142 of the two